src/components/panel/panel.spec.js

```diff
@@ -122,6 +122,36 @@ describe('$mdPanel', function() {
     expect(PANEL_WRAPPER_CLASS).not.toHaveClass(HIDDEN_CLASS);
   });
 
+
+  it('should destroy the panel when closing', function () {
+    openPanel(DEFAULT_CONFIG);
+
+    expect(panelRef._panelContainer).not.toHaveClass(HIDDEN_CLASS);
+    expect(panelRef.isAttached).toEqual(true);
+    expect(panelRef._config.locals).not.toEqual(null);
+
+    panelRef.close();
+
+    $rootScope.$apply();
+
+    expect(panelRef.isAttached).toEqual(false);
+    expect(panelRef._config.locals).toEqual(null);
+  });
+
+  it('should focus on origin element after hide (if provided)', function () {
+    attachToBody('<button id="donuts">Donuts</button>');
+
+    var detachFocusConfig = angular.extend({ origin: '#donuts' }, DEFAULT_CONFIG);
+
+    openPanel(detachFocusConfig);
+
+    panelRef.hide();
+
+    $rootScope.$apply();
```

Comment:
It would seem this apply is still necessary.. Unfortunately the tests fail without it :(